marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -832,7 +911,7 @@ const Flows: React.FC<Props> = (props) => {
     try {
       let response = await axios.get("/api/jobs/" + step.jobId);
       if (response.status === 200) {
-        props.showStepRunResponse(step.jobId);
+        props.showStepRunResponse(step, step.jobId, response.data);
```

Comment:
I think the showStepRunResponse just receives a step as a prop

Reply:
Updated based in the commit [8541](https://github.com/marklogic/marklogic-data-hub/pull/6585/commits/d33093006c92e72d61460f67fb6a742481d31a41)